pntr.h

```diff
@@ -600,6 +598,23 @@ PNTR_API void pntr_unload_memory(void* pointer);
 PNTR_API void* pntr_memory_copy(void* destination, void* source, size_t size);
 PNTR_API pntr_image_type pntr_get_file_image_type(const char* filePath);
 
+// thick
```

Comment:
Looking good! This may end up being in doxygen, so we'll leave the hints out for now :+1: 
``` suggestion
```